data/schemas/com.m3u.data.database.M3UDatabase/16.json

```diff
@@ -2,7 +2,7 @@
   "formatVersion": 1,
   "database": {
     "version": 16,
-    "identityHash": "5ef08bb6389682643fa285d418ec2958",
```

Comment:
You need to upgrade and migrate the database instead of changing the tables directly.

Reply:
Or the users who upgrade from old versions will lose all their data.

Reply:
this is correct hash right? "identityHash": "6c929023e487d71831acf3de62d3389b",

Reply:
No 16.json content should not be changed. It is updated that because you didn't upgrade the db version but change the table and then build or run the project.

Reply:
Rollback the json and do not build or run project, upgrade db version number to 17 then you can build or run, then the room compiler will generate the new 17.json